packages/cmf/src/cmfConnect.js

```diff
@@ -42,6 +44,40 @@ import { mapStateToViewProps } from './settings';
 
 let newState;
 
+const KEEP_EVENT_ATTRIBUTES = [
+	'type',
+	// keyboard
+	'ctrlKey',
+	'altKey',
+	'shiftKey',
+	'charCode',
+	'key',
+	'keyCode',
+	'repeat',
+	'which',
+	// mouse
+	'button',
+	'buttons',
+	'clientX',
+	'clientY',
+	'pageX',
+	'pageY',
+	'screenX',
+	'screenY',
+];
+
+function serializeEvent(event) {
+	if (event.nativeEvent) {
+		return Object.assign(
+			{
+				target: pick(event.target || {}, ['type', 'value', 'checked']),
+			},
+			pick(event, KEEP_EVENT_ATTRIBUTES),
```

Comment:
Is this really needed?

Reply:
look at event pooling: https://reactjs.org/docs/events.html#event-pooling

Reply:
Ok 👌